perun-web-gui/src/main/java/cz/metacentrum/perun/webgui/tabs/facilitiestabs/FacilitySettingsTabItem.java

```diff
@@ -121,10 +125,30 @@ public Widget draw() {
 
 		// Get Attributes method
 		final GetRequiredAttributesV2 reqAttrs = new GetRequiredAttributesV2();
-
 		// get empty table
 		final CellTable<Attribute> table = reqAttrs.getEmptyTable();
 
+		final GetAttributesV2 attrs = new GetAttributesV2(new JsonCallbackEvents(){
+			@Override
+			public void onFinished(JavaScriptObject jso){
+				List<Attribute> list = JsonUtils.<Attribute>jsoAsList(jso);
+				for(int i = 0; i < list.size(); i++){
+					reqAttrs.addToTable(list.get(i));
```

Comment:
Please use ``setList();`` instead of ``addToTable()`` and then call ``sortTable();``. Thanks.